sdk/react/controlBar.js

```diff
@@ -185,7 +185,7 @@ class ControlBar extends React.Component {
         iconTouchableStyle: styles.iconTouchable,
         durationString: this.getDurationString()
       },
-      fullscreen: {
+      Fullscreen: {
```

Comment:
Why do we need this capitalized?

Reply:
In shortly, because we have FULLSCREEN: 'Fullscreen' in constants.js (BUTTON_NAMES) and that value is used in a iOS and Android bridges. 
I a long term , because we need aligned button names in all parts : bridge, react native code, skin.json. And i just started from "fullscreen" button because fix is need it to be aligned.